### PR TITLE
docs(tutorial): add tutorial how to use your own grafana

### PR DESCRIPTION
**Pull request by nedaehm**

### CODE REVIEW

File: tutorials/using-own-grafana/index.mdx [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+---
+
+In this tutorial, you will learn how to use your own Grafana with [Scaleway's Cockpit](/observability/cockpit/quickstart/) service. In this tutorial we are using Grafana version 9.5.1.
+
+<Macro id="iam-requirements" />
+
+<Message type="requirement">
+  - You have an account and are logged into the [Scaleway console.](https://console.scaleway.com)
+  - You have [activated your Cockpit](/observability/cockpit/how-to/activate-cockpit/)
+  - You have [installed docker](https://docs.docker.com/engine/install/)
```
Comment (bene2k1) [inline tutorials/using-own-grafana/index.mdx:24]:
```suggestiond
  - You have [installed Docker](https://docs.docker.com/engine/install/)
```

File: tutorials/using-own-grafana/index.mdx [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+  - You are using the [latest version of Grafana](https://grafana.com/grafana/download)
+</Message>
+
+## Launching Grafana
+
+1. Run the following command to launch Grafana on your local machine:
```
Comment (bene2k1) [inline tutorials/using-own-grafana/index.mdx:30]:
```suggestion
1. Run the following command to launch Grafana in a Docker container on your local machine:
```

File: tutorials/using-own-grafana/index.mdx [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+meta:
+  title: Using your own Grafana with Cockpit
+  description: This page shows how to use your own Grafana using Scaleway Cockpit.
+content:
+  h1: Using your own Grafana with Cockpit
+  paragraph: This page shows how to use your own Grafana using Scaleway Cockpit.
+categories:
+  - observability
+  - cockpit
+tags: cockpit metrics logs alerts prometheus loki grafana
```
Comment (TuTuRyYE) [inline tutorials/using-own-grafana/index.mdx:11]: add mimir ?

File: tutorials/using-own-grafana/index.mdx [hunk 4]
```diff
@@ -0,0 +1,138 @@
...
+2. [Create a token](/observability/cockpit/how-to/create-token/) with the following permissions:
+    - Query metrics
+    - Query logs
+    - Rules for metrics
+    - Rules for logs
+    - Alerts
+     Your token configuration should look like the following:
+      <Lightbox src="scaleway_cockpit_tuto.webp" alt="" />
+3. Copy your token's secret key as you will need it for the following steps.
+4. Click the [following link](http://localhost:3000) to access Grafana.
```
Comment (bene2k1) [inline tutorials/using-own-grafana/index.mdx:44]:
```suggestion
4. Open [http://localhost:3000](http://localhost:3000) in a web browser on your local computer to access Grafana.
```

File: tutorials/using-own-grafana/index.mdx (hunk 3 above)
Comment (nedaehm) [reply tutorials/using-own-grafana/index.mdx]: Done 👍🏼

**Review by bene2k1 — APPROVED** [review]

File: tutorials/using-own-grafana/index.mdx [hunk 5]
```diff
@@ -0,0 +1,138 @@
...
+categories:
+  - observability
+  - cockpit
+tags: cockpit metrics logs alerts prometheus loki grafana mimir 
+dates:
+  validation: 2023-05-10
+  posted: 2023-05-10
+---
+
+In this tutorial, you will learn how to use your own Grafana with [Scaleway's Cockpit](/observability/cockpit/quickstart/) service. In this tutorial we are using Grafana version 9.5.1.
```
Comment (ldecarvalho-doc) [inline tutorials/using-own-grafana/index.mdx:17]:
```suggestion
In this tutorial, you will learn how to use your own Grafana with [Scaleway's Cockpit](/observability/cockpit/quickstart/) service. In this tutorial, we are using Grafana version 9.5.1.
```

**Review by ldecarvalho-doc — APPROVED** [review]

**Pull request merged**